specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-04-01/mfe.json

```diff
@@ -19850,6 +20871,26 @@
       },
       "additionalProperties": false
     },
+    "PartialMinimalTrackedResourceWithSkuAndIdentity": {
+      "additionalProperties": false,
+      "allOf": [
+        {
+          "$ref": "#/definitions/PartialMinimalTrackedResource"
+        }
+      ],
+      "description": "Strictly used in update requests.",
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/PartialManagedServiceIdentity",
```

Comment:
our sister team (that usually manages the API releases for all MachineLearning services) has work items planned to do this migration in a future PR. since this PR has already been merged, I don't think this PR is a good time to address these changes